ophyd/utils/epics_pvs.py

```diff
@@ -384,15 +406,26 @@ def set_and_wait(signal, val, poll_time=0.01, timeout=10):
                                (signal, val, timeout, current_value))
 
 
-def _compare_maybe_enum(a, b, enums):
-    if not enums:
+def _compare_maybe_enum(a, b, enums, atol, rtol):
+    if enums:
+        # convert enum values to strings if necessary first:
+        if not isinstance(a, str):
+            a = enums[a]
+        if not isinstance(b, str):
+            b = enums[b]
+        # then compare the strings
         return a == b
-    if not isinstance(a, str):
-        a = enums[a]
-    if not isinstance(b, str):
-        b = enums[b]
+
+    # if either relative/absolute tolerance is used, use numpy
+    # to compare:
+    if atol is not None or rtol is not None:
+        return np.allclose(a, b,
+                           rtol=rtol if rtol is not None else 1e-7,
```

Comment:
Guess I got confused. `numpy` mixes them up too on occasion:
```python
def allclose(a, b, rtol=1.e-5, atol=1.e-8):
def assert_allclose(actual, desired, rtol=1e-7, atol=0, err_msg='', verbose=True):
```

Worth it for 2 lines? Or you want these defaults passed in?